src/index.js

```diff
@@ -49,11 +50,26 @@ async function main () {
     const config = await getFullConfig();
     const verbosity = getVerbosity(config.verbosity);
     const logger = new Logger(verbosity);
-    const { GITHUB_HEAD_REF, GITHUB_BASE_REF, GITHUB_REPOSITORY } = process.env;
-    config.urlRewriteExpressions.push({
-      pattern: new RegExp(`(github.com/${GITHUB_REPOSITORY}/.*/)(${GITHUB_BASE_REF})/(.*)`),
-      replacement: `$1${GITHUB_HEAD_REF}/$3`
-    });
+    const { GITHUB_HEAD_REF, GITHUB_BASE_REF, GITHUB_REPOSITORY, GITHUB_EVENT_PATH } = process.env;
+    // Read pull_request payload and use it to determine head user/repo:
+    if (GITHUB_EVENT_PATH) {
+      try {
+        const payloadRaw = await readFile(GITHUB_EVENT_PATH, 'utf8');
+        const payload = JSON.parse(payloadRaw);
+        if (payload?.pull_request?.head) {
```

Comment:
This is annoying, but could I think GitHub Actions built this way are stuck on node.js 12, where optional chaining won't work.  I should probably use Node 12 in my CI.